### PR TITLE
link to process instead of onClick

### PR DESCRIPTION
**Pull request by JulianWielga**
allows opening process in new window

### CODE REVIEW

File: ui/client/containers/SubProcesses.tsx [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
 import tabStyles from "../components/tabs/processTabs.styl"
 import {SearchItem} from "./TableFilters"
 
 function ShowProcessIcon({process}: {process: ProcessType}) {
-  const dispatch = useDispatch()
-  const onClick = useCallback(() => dispatch(showProcess(process.name)), [process.name])
   return (
-    <TableRowIcon glyph="edit" title="Edit subprocess" onClick={onClick}/>
+    <ProcessLink processId={process.name}>
+      <TableRowIcon glyph="edit" title="Edit subprocess"/>
```
Comment (lciolecki) [inline ui/client/containers/SubProcesses.tsx:17]: Missing translation

File: ui/client/containers/editItem.tsx [hunk 2]
```diff
@@ -1,27 +1,20 @@
...
 export function EditItem({process}: {process: ProcessType}) {
-  const dispatch = useDispatch()
-  const clickHandler = useCallback(
-    () => dispatch(showProcess(process.name)),
-    [process],
-  )
   return (
-    <TableRowIcon glyph="edit" title="Edit process" onClick={clickHandler}/>
+    <ProcessLink processId={process.name}>
+      <TableRowIcon glyph="edit" title="Edit process"/>
```
Comment (lciolecki) [inline ui/client/containers/editItem.tsx:9]: Missing translation

File: ui/client/.eslintrc [hunk 3]
```diff
@@ -212,7 +212,8 @@
           "cn",
           "api.get",
           "api.post",
-          "describe"
+          "describe",
+          "css"
```
Comment (lciolecki) [inline ui/client/.eslintrc:216]: Missing comma after "css"

Comment (JulianWielga) [reply ui/client/.eslintrc:216]: not missing, its illegal in json

**Pull request merged**